core/input/input_map.cpp

```diff
@@ -400,6 +400,7 @@ static const _BuiltinActionDisplayName _builtin_action_display_names[] = {
     { "ui_filedialog_refresh",                         TTRC("Refresh") },
     { "ui_filedialog_show_hidden",                     TTRC("Show Hidden") },
     { "ui_swap_input_direction ",                      TTRC("Swap Input Direction") },
+	{ "ui_unicode_start",                              TTRC("Start Unicode Character Input") },
```

Comment:
```suggestion
    { "ui_unicode_start",                              TTRC("Start Unicode Character Input") },
```

Or re-indent the whole block with a tab, I don't know why it was intended with 4 spaces (clang-format is disabled to keep the 2nd column aligned).